src/plugins/ui_actions/public/components/error_configure_action/i18n.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const txtSorryActionConfigurationError = i18n.translate(
```

Comment:
why are we exporting all strings to separate file ? is this a new convention i didn't hear about ?

Reply:
I've put the translation statements into a separate file because it is convenient, otherwise it clutters the React component and makes it harder to read. I wasn't guided by any convention, just trying to make code readable.

But in general in Kibana there are many places where translations are separated out in a standalone file, for example, Canvas has a whole folder `x-pack/legacy/plugins/canvas/i18n` with a dozen of files.